src/esw/brushless.py

```diff
@@ -221,16 +229,11 @@ def __init__(self):
         self.drive_bridge_by_name = {}
         using_pi3_hat = rospy.get_param("brushless/using_pi3_hat")
         if using_pi3_hat:
-            # import moteus_pi3hat
-            # transport = moteus_pi3hat.Pi3HatRouter(
-            #     servo_bus_map={
-            #         1: [11],
-            #         2: [12],
-            #         3: [13],
-            #         4: [14],
-            #     },
-            # )
-            pass
+            import moteus_pi3hat
```

Comment:
as a result of this import statement, will i still be able to run this program if i use fdcanusb and set using_pi3_hat to false in the config? according to internet this is valid but it looks unfamiliar to me so just double checking lol

Reply:
answer is yes